### ISSUE TITLE
v3 Update

### ISSUE THREAD

**Issue by aquarion** [dustin/py-github#64]
With @Github's news that they're turning off v2 in a couple of months, are you planning to update this to v3? (Query, rather than passive-aggressive bug. I'm trying to work out if I should start migrating)

**Comment by dustin** [comment]
I think it's a good bug.  I haven't been following that direction very much, but I do use tools from here, so getting it updated would be beneficial to me.  I haven't started any work in this direction yet, though (other than what's trickled in).

**Comment by jtriley** [comment]
+1

**Comment by MikeEsteves** [comment]
+2

**Comment by aquarion** [comment]
So, er, I assume your stuff has just stopped working too?

**Comment by dustin** [comment]
Yep, it seems to have.  :/  I have entirely too many projects.  This one has parts I've used, but can't now.  There were a couple of efforts from contributors to move towards an API v3 client, but I haven't seen a pull request, so I should turn my hopes down a bit.

Does anyone out there want to help make progress?  I certainly use these APIs myself.

**Comment by sigmavirus24** [comment]
I hope this doesn't come as an unwelcome comment. I saw that @sethwoodworth starred my [sigmavirus24/github3.py](https://github.com/sigmavirus24/github3.py) project which is an attempt at providing an interface to the v3 API. From looking at a bit of your README I would say that mine isn't going to provide an easy transition from your library to mine but there is an (unfortunately incomplete) library [copitux/python-github3](https://github.com/copitux/python-github3) which seems to have a similar interface to yours.

If anyone starts using mine and finds issues/bugs, a pull request is always welcome. I only have 87% coverage (including calls that require you to authenticate) on tests. The only thing github3.py is missing is the distinction between stargazers and subscribers (watchers).

## 

Ian
